CONTRIBUTING.md

```diff
@@ -129,6 +129,21 @@ When you have set up your fork of a repository that you want to contribute to an
 4. **Validating and verifying the PR:** 
    - Reviewers can review the pull request on the GitHub website or fetch the PR using the _Fetch GitHub PR_ menu entry in the _Git Repositories_ view inside the Eclipse IDE.
    - Every PR is automatically verified to check that contributors have a [valid Eclipse Contributor Agreement (ECA)](#setting-up-your-eclipse-and-github-account).
+   ### What does a valid PR look like? (Checklist)
+   These are the basic criteria that need to be fulfilled for a PR to be considered valid. 
+   
+   **If you are a contributor, consider these points as prerequisites before your PR can be reviewed. Please ask for help if needed, either by making a comment on the PR or via the [dev mailing list](#contact).**
```

Comment:
I have just one personal remark, we have actually enabled github discussions enabled everywhere and I think that's more suitable to mention than the dev-mailing list for asking questions.

Reply:
@laeubi I see your point and I partially agree: I personally prefer GH discussions over the mailing list, but the mailing list is still the "official" way so I wouldn't remove it. 

How about:
```
**If you are a contributor, consider these points as prerequisites before your PR can be reviewed. 
Please ask for help if needed, either by making a comment on the PR or via a _GitHub discussion_ or the [dev mailing list](#contact).**
```

?

@jonahgraham since you mentioned the list, I would like your opinion on this too.

Reply:
The mailinglist is only required for official announcements, not for user questions.

Reply:
I think "we all" really don't want email questions.   We already get emails about discussions and with that approach one can focus on collecting the information in the discussion rather than a bunch of divergent email threads.

Reply:
> The mailinglist is only required for official announcements, not for user questions.

> I think "we all" really don't want email questions. We already get emails about discussions and with that approach one can focus on collecting the information in the discussion rather than a bunch of divergent email threads.

Ok, the majority seems to prefer the GH discussions over the e-mails so I just removed the mention of the e-mails in 8a47991e3f40f1f3fdb8cfd3ba29b9c5e5682fa8. @jonahgraham fyi